sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -22,28 +22,40 @@ namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals
     /// </remarks>
     internal partial class Manager
     {
-        private Action _callbackAction = () => { };
+        private Thread? _backgroundThread;
+        private static readonly ManualResetEvent _shutdownEvent = new(false);
```

Comment:
Why is `_backgroundThread` an instance thing but `_shutdownEvent` is static? Down below in `Dispose` we dispose both. Kind of odd to see an instance of something disposing a static thing. My recommendation would be drop `static` from `_shutdownEvent`.